content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -613,6 +613,249 @@ To test it, change the context to `myuser`:
 kubectl config use-context myuser
 ```
 
+<!-- TODO this should become a task page -->
```

Comment:
Can we make this a task page before the first merge?

Reply:
Yes totally - I didn't know what the preference was, since the other set above was not a task yet. Do you think we should put this under `content/en/docs/tasks/administer-cluster/`?